research/2022-08-20-introducing-waku-product.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+name:  "Waku Product"
+title:  "Waku Product"
+date:   2022-08-20 00:00:00 +0000
+author: franck
+published: true
+permalink: /waku-product
+categories: waku, product, platform, operator
+summary: Introduce Waku Product, a project to ensure that Waku is usable and used.
+image: /img/black-waku-logo-with-name.png
+discuss: TODO
+---
+
+# Waku for Web DApps
+
+In 2021, we started to push for the adoption of Waku by projects other than the Status app.
+Waku is the communication component of the Web3 trifecta,
+which originally was Ethereum (contracts), Swarm (storage) and Whisper (communication).
+Hence, it made sense to first target dApps which already uses one of the Web3 pillar: Ethereum. 
+
+As most dApps are web apps,
+we naturally started the development of [js-waku for the browser](https://vac.dev/presenting-js-waku).
+
+Once ready,
+we reached out to dApps to integrate Waku,
+added [prizes to hackathons](https://twitter.com/waku_org/status/1451400128791605254?s=20&t=Zhc0BEz6RVLkE_SeE6UyFA)
+and gave [talks](https://docs.wakuconnect.dev/docs/presentations/).
+
+We also assumed that we would see patterns in the usage of Waku,
+that we would facilitate with the help of [SDKs](https://github.com/status-im/wakuconnect-vote-poll-sdk).
+
+Finally, we created a number of web apps:
+[examples](https://docs.wakuconnect.dev/docs/examples/)
+and [PoCs](https://github.com/status-iM/gnosis-safe-waku).
+
+We branded this overall web app outreach effort as Waku Connect:
```

Comment:
Who is the audience for this post? Do people care that we previously branded it as this?